index.html

```diff
@@ -0,0 +1,290 @@
+          <li>
+            <a class="social-item" href="#">
+              <img src="images/Angelist.png" alt="Angelist"> </a>
+            </a>
+          </li>
+          <li>
+            <a class="social-item" href="#">
+              <img src="images/Twitter.png" alt="twitter"> </a>
+            </a>
+          </li>
+          <li>
+            <a class="social-item" href="#">
+              <img src="images/Medium.png" alt="medium"> </a>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <section class="works">
+      <h2 class="my-recent">My Recent Works</h2>
+      <hr />
+      <div class="placeholder">
+        <img src="images/Img_Placeholder.png" alt="Image Placeholder" />
+      </div>
+      <div class="stories">
+        <p>Multi-Post Stories</p>
+      </div>
+      <p class="daily">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required. has been the industry's standard dummy text ever
+        since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
+      <ul class="lruby">
+        <a class="block" href="#">
+          <li>css</li>
+        </a>
+        <a class="block" href="#">
+          <li>html</li>
+        </a>
+        <a class="block" href="#">
+          <li>bootstrap</li>
+        </a>
+        <a class="block" href="#">
+          <li>ruby</li>
+        </a>
+      </ul>
+      <a class="see-project" href="#"> See Project </a>
+    </section>
+
+    <section class="project-cards">
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
+      <div class="profesion">
+        <h2 class="heading-content">
+          Profesional Art <br />
+          Printing Data
+        </h2>
+        <p class="p-content">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="p-language">
+          <a class="block-two" href="#">
+            <li>html</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>bootstrap</li>
+          </a>
+          <a class="block-two" href="#">
+            <li>ruby</li>
+          </a>
+        </ul>
+        <button class="project-button" href="#">See Project</button>
+      </div>
```

Comment:
I suggest adding background images just as the figma design has. For this project, part of the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) was to stick as much as possible to the design
<img width="1433" alt="Screenshot 2022-12-02 at 14 39 40" src="https://user-images.githubusercontent.com/48546992/205285010-84730a18-9b1d-47d3-9c06-639333727785.png">